Mergin/plugin.py

```diff
@@ -324,6 +331,38 @@ def configure_db_sync(self):
         if not wizard.exec_():
             return
 
+    def switch_sources(self):
+        project_path = QgsProject.instance().homePath()
+        if not project_path:
+            iface.messageBar().pushMessage("Mergin", "Project is not saved, please save project first", Qgis.Warning)
+            return
+
+        if not check_mergin_subdirs(project_path):
+            iface.messageBar().pushMessage(
+                "Mergin", "Current project is not a Mergin project. Please open a Mergin project first.", Qgis.Warning
+            )
+            return
+
```

Comment:
worth adding a `unsaved_project_check()` test here as well - like in view_local_changes()